doc-content/enterprise-only/openshift/template-deploy-nopubnet-s2i-proc.adoc

```diff
@@ -0,0 +1,10 @@
+[id='template-deploy-nopubnet-s2i-{context}-proc']
+= Configuring access to a Maven mirror to perform an S2I build without a connection to the public Internet
+
+If your OpenShift environment does not have a connection to the public Internet, you can enable an S2I build using a Maven mirror that you set up according to <<offline-repo-proc>>.
+
+.Procedure
```

Comment:
Good point, but prereqs at this late stage would take a SME review that is just not available. This entire part will be revamped for 7.3.1 anyway, as the functionality will be improved for 7.3.1 based on my work - which took too long for 7.3.0 because I faced quite a few hurdles of a technical kind, and for some of them our Dev did not have solutions and I eventually found them myself.